### ISSUE TITLE
[FEATURE] Handakte inhaltlich erweitern

### ISSUE THREAD

**Issue by Dickmaulruessler** [jlawyerorg/j-lawyer-org#1546]
Da #1544 und da #1545 habe ich gesehen, dass selbst kleinste Änderungen ganz schön kompliziert sind in der Programmierung.... Puhhh! Daher habe ich bei nachfolgendem Vorschlag schon ein schlechtes Gewissen. 

Ich nutze die **"Handakte"** bisher nicht, da sie mir einfach zu wenig Informationen enthält. Ich könnte mir hierfür vorstellen, dass diese umfangreicher wird. Für einen schnellen Überblick auf Papier wäre für mich hilfreich

- eine **komplette Dokumentenliste** (Datum, Dateiname) und 
- die **"Notizen" im Volltext**. 

Ich weiß allerdings, dass dies je nach Nutzung auch sehr unpraktikabel sein könnte: Manche Kolleginnen und Kollegen schreiben sehr viel in die Notizen rein. Bei mir stehen da immer nur sehr wenige Sätze. Meine Akten sind auch nie so umfangreich, manche haben da vielleicht sehr viele Dateien. Da könnte mein Vorschlag schnell unpraktikabel werden. Es sei denn, man könnte die Inhalte der Handakte auswählbar machen. Aber das macht die Sache wahrscheinlich noch kompliziert.... 
Naja, ich stelle das einmal zur Diskussion... und hätte Verständis dafür, falls das gleich wieder geschlossen wird....

**Comment by Dickmaulruessler** [comment]
Je länger ich drüber nachdenke, desto praktischer würde ich es finden, im Termin die **komplette Dokumentenliste** in Papierform zu haben _(Bei meinen Terminen im Betreuungsrecht wäre das genau das richtige. Ich hätte dann nur dies und das letzte Gutachten, welches ich vom Gericht ohnehin nur in Papierform bekomme, dabei)._

Könnte ich die Liste irgendwie nerdig provisorisch selbst erstellen, also diese Dokumentenliste manuell ausdrucken? Mir fiele jetzt nur ein Screenshot ein, was ich wiederum zu primitiv finde...

**Comment by Dickmaulruessler** [comment]
Mal in die Runde gefragt: 

Benötigt irgendjemand das "Kostenvorblatt" in der Handakte? 
Das ist praktisch nichts anderes als eine Kopiervorlage (mit eingefügtem Aktenzeichen).
Könnt ihr euch irgendjemanden vorstellen, der das (nur handschriftlich ausfüllbare) Blatt gebrauchen könnte?

Ich schlage dessen Entferung vor.

**Comment by Fn0rdmann** [comment]
Ja, ich benutze das gelegentlich. Wer es nicht braucht, kann es beim Ausdruck einfach weglassen. Wer gar nicht mehr druckt, kann es einfach ignorieren.

**Comment by j-dimension** [comment]
> Könnte ich die Liste irgendwie nerdig provisorisch selbst erstellen, also diese Dokumentenliste manuell ausdrucken? Mir fiele jetzt nur ein Screenshot ein, was ich wiederum zu primitiv finde...

Akte als HTML exportieren und dann die HTML-Datei über den Browser drucken? Allerdings auch irgendwie nur so semi-professionell :-)
